docs/markdown/FAQ.md

```diff
@@ -683,3 +683,21 @@ executable(
   link_language : 'c',
 )
 ```
+
+## How do I use `include-what-you-use` with meson?
+
+Following the steps from
+[Using with a compilation database](https://github.com/include-what-you-use/include-what-you-use#using-with-a-compilation-database),
+configure the build directory (e.g., with `meson setup build`) and run:
+
+```console
+$ /path/to/iwyu_tool.py -p build
+```
+
+See `--help` for more options.
+
+A run target can also be added with the following meson code:
+```meson
+iwyu_tool = find_program('iwyu_tool.py')
```

Comment:
I don't think `.py` exists if you install the package for instance. @xclaesse might be nice to be able to specify multiple program names similar to multiple dependency names for times like this.

Reply:
it seems to be installed as `iwyu-tool`. updated the patch

Reply:
find_program() has always accepted multiple names ;-)